tests/ndarray/test_lazyexpr.py

```diff
@@ -1349,8 +1349,50 @@ def test_chain_expressions():
     le1 = a**3 + blosc2.sin(a**2)
     le2 = le1 < c
     le3 = le2 & (b < 0)
-
     le1_ = blosc2.lazyexpr("a ** 3 + sin(a ** 2)", {"a": a})
     le2_ = blosc2.lazyexpr("(le1 < c)", {"le1": le1_, "c": c})
     le3_ = blosc2.lazyexpr("(le2 & (b < 0))", {"le2": le2_, "b": b})
     assert (le3_[:] == le3[:]).all()
+
+    # TODO: This test should pass eventually
+    # le1 = a ** 3 + blosc2.sin(a ** 2)
+    # le2 = le1 < c
+    # le3 = (b < 0)
+    # le4 = le2 & le3
+    # le1_ = blosc2.lazyexpr("a ** 3 + sin(a ** 2)", {"a": a})
+    # le2_ = blosc2.lazyexpr("(le1 < c)", {"le1": le1_, "c": c})
+    # le3_ = blosc2.lazyexpr("(b < 0)", {"b": b})
+    # le4_ = blosc2.lazyexpr("(le2 & le3)", {"le2": le2_, "le3": le3_})
+    # assert (le4_[:] == le4[:]).all()
+
+
+# TODO: Test the chaining of multiple persistent lazy expressions
+# def test_chain_persistentexpressions():
```

Comment:
Remember to change this: `test_chain_persistentexpressions` -> `test_chain_persistent_expressions`. As a general rule, you must separate complete words with underscores.  Also, 4 words should be a hard limit for names ;-)